README.md

```diff
@@ -133,8 +133,8 @@ suffixed with the right ports for the various services. You can of course create
 	- Check _`Use settings from uploaded test plan` check box bellow `Jmeter 3.0` tool
```

Comment:
Missing an underscore here. Also make sure JMeter is always spelled as "JMeter".